code/src/java/pcgen/core/Equipment.java

```diff
@@ -2349,19 +2349,6 @@ public String getWieldName()
 	}
 
 	/**
-	 * Description of the Method
-	 * 
-	 * @param aPC The PC that has this Equipment
-	 * 
-	 * @return Description of the Return Value
-	 */
-	public Integer acCheck(final PlayerCharacter aPC)
-	{
-		return Math.min(getSafe(IntegerKey.AC_CHECK)
-			+ (int) bonusTo(aPC, "EQMARMOR", "ACCHECK", true), 0);
```

Comment:
I'm operating with a bias to delete methods from Equipment and PlayerCharacter when possible to simplify those interfaces, and prevent usage of the old methods elsewhere... force as much to the (old, to be deprecated) output system and terms as possible